tool/gravity/cli/clusterupdate.go

```diff
@@ -217,6 +217,14 @@ func executeUpdatePhase(env *localenv.LocalEnvironment, environ LocalEnvironment
 }
 
 func executeUpdatePhaseForOperation(env *localenv.LocalEnvironment, environ LocalEnvironmentFactory, params PhaseParams, operation ops.SiteOperation) error {
+	allActive, err := verifyActiveAgents(env)
```

Comment:
Also, I don't think this check should be placed here. This function is invoked for every manual phase invocation e.g. using "gravity plan execute" command. I think this check should only be executed when a user does "gravity resume" or "gravity rollback".

Reply:
54103b0 Does this look right?

`gravity plan resume` can still be executed if an agent is offline. `gravity upgrade resume` and now `gravity rollback` will fail if an agent is offline.